deploy/kubernetes/overlays/dev/kustomization.yaml

```diff
@@ -5,3 +5,4 @@ bases:
 images:
 - name: amazon/aws-ebs-csi-driver
   newTag: latest
+  newName: chengpan/aws-ebs-csi-driver
```

Comment:
Was this intentional?

Reply:
yep. since it is under dev overlay and I feel it could be handy for local testing. Let me know if you think otherwise